### DISCUSSION TITLE
Unable to add multiple lights in .yaml file

### DISCUSSION THREAD

**Discussion by XavChris** [sbidy/wiz_light#127] [Q&A]
So I have all my bulb names and IP addresses 
- platform: wiz_light
name: "Office 1"
host: 192.168.86.27
- platform: wiz_light
name: “Christa’s Lamp”
host: 192.168.86.26
- platform: wiz_light
name: “Xavier’s Lamp"
host: 192.168.86.23
- platform: wiz_light
name: “Office 2”
host: 192.168.86.32
- platform: wiz_light
name: “Wall 2”
host: 192.168.86.46
- platform: wiz_light
name: “Wall 1”
host: 192.168.86.20

but the .yaml file errors out

end of the stream or a document separator is expected at line 5, column 1:
    - platform: wiz_light
    ^
If I only put one light it works fine

Can anyone point me in the correct direction?

Thanks in advance!

**Comment by jcasarini** [comment]
Can you upload a screenshot of the configuration.yaml (cut this part only, redact any sensitive information), there's probably an indentation problem in the file, but it won't show in what you pasted,
